Add option for surface pH output

JorgSchwinger opened the pull request:
Has been tested with and without natDIC option.

JorgSchwinger committed:
Add option for output of surface pH

JorgSchwinger committed:
Correct accumulation of natural hi

JorgSchwinger committed:
Correct white spaces

jmaerz reviewed (approved):
Looks generally ok to me. The question is generally, if you want to generate the time mean ph, or ph of time mean H+ concentration, though (which can make a -likely subtle- difference). Only minor thing: one could rename 'h+' to 'H+' in the netcdf variable units (as in '-log10(H+)').

TomasTorsvik reviewed (approved):
Thanks, @JorgSchwinger , no complaints from my side!

JorgSchwinger committed:
Changed h+ to H+ in netCDF output of pH units

JorgSchwinger commented:
Thanks, Jöran and Tomas for reviewing this quickly!

@jmaerz I also just got aware that we do not save the average pH, but the log10 of the average H+ concentration. Since this has been done always like this for the 3d-pH field in HAMOCC (at least in Bergen), I implemented it consistent with the 3d fields. I am not sure how large the difference is. If there is a difference, one should have the option for both average H+ and average pH.

I have changed h+ to H+.
